### PR TITLE
Image descriptions for 7.3

### PR DESCRIPTION
**Pull request by supervladimir**

### CODE REVIEW

File: ptx/sec_shell_method.ptx [hunk 1]
```diff
@@ -107,16 +108,16 @@
...
-                  The entire region inside this shape is solid.
-                </p>
-            </description>
-            <shortdescription>A 3-d graph of the function in the first quadrant rotated fully about the y-axis.</shortdescription>
+          <description>
+            <p>
+              A three dimensional diagram of the area enclosed by the curve <m>y=\frac{1}{1+x^2} </m>, and the lines <m>x=0 </m>, <m> x=1 </m> and <m>y=0 </m> rotated about the <m>y</m>-axis.
+              The base of this shape is a unit circle lying on the <m>xz</m> plane.
+              This unit circle comes up from <m>y=0</m> to <m>y=0.5</m> creating a solid cylinder.
+              The region above <m>y=0.5</m> forms a shape which looks a perfectly symmetrical mountiain or hill, whose peak is the point <m>(0,1,0)</m>.
```
Comment (sean-fitzpatrick) [inline ptx/sec_shell_method.ptx]: - looks like (missing word)
- typo: mountain

File: ptx/sec_shell_method.ptx [hunk 2]
```diff
@@ -330,7 +331,17 @@
       <figure xml:id="fig_soupcana_3D">
         <caption/>
         <image xml:id="img_shell_soupcan">
-          <description></description>
+          <description>
+           <p>
+              The left side of the image contains a cylinder, and the right side of the image shows a rectangle which comes from unraveling the side of the cylinder.
+              The cylinder has a height <m> h </m>, radius <m> r </m> and is pictured lying on it's circular base.
```
Comment (sean-fitzpatrick) [inline ptx/sec_shell_method.ptx]: should be "its" and not "it's"

File: ptx/sec_shell_method.ptx [hunk 3]
```diff
@@ -330,7 +331,17 @@
       <figure xml:id="fig_soupcana_3D">
         <caption/>
         <image xml:id="img_shell_soupcan">
-          <description></description>
+          <description>
+           <p>
+              The left side of the image contains a cylinder, and the right side of the image shows a rectangle which comes from unraveling the side of the cylinder.
+              The cylinder has a height <m> h </m>, radius <m> r </m> and is pictured lying on it's circular base. 
+              Both the circular base, and the top of the cylinder are shaded, while the curved surface is unshaded, but contains a "cut here" instruction.
```
Comment (sean-fitzpatrick) [inline ptx/sec_shell_method.ptx]: there are xml tags for quotes (so they get handled correctly in LaTeX, etc: use `<q>cut here</q>`

File: ptx/sec_shell_method.ptx [hunk 4]
```diff
@@ -365,7 +376,18 @@
       <figure xml:id="fig_soupcanb_3D">
         <caption/>
         <image xml:id="img_shell_unwrapshell">
-          <description></description>
+          <description>
+            <p>
+              The left side of the image contains a cylindrical shell, which is just a cylinder with the middle part cut out. 
+              The right side of the image shows a rectangular box which comes from unraveling the cylindrical shell.
+              The cylindrical shell has a height <m> h </m>, radius <m> r </m> which spans from the cut out center to the edge of the shell, and the shell has thickness <m> \dx </m> and is pictured lying on it's circular base.
```
Comment (sean-fitzpatrick) [inline ptx/sec_shell_method.ptx]: same comments: "its", and the quotes.

File: ptx/sec_shell_method.ptx [hunk 5]
```diff
@@ -467,7 +489,18 @@
...
-          <description></description>
+          <description>
+            <p>
+              Graph of the region bounded by <m>y=0</m>, <m>y=1/(1+x^2)</m>, <m>x=0</m> and <m>x=1</m>. 
+              The curve <m>y=1/(1+x^2)</m> begins at the <m> y</m>-axis at point <m> (0,1) </m> and slopes downwards before ending at the point <m> (1,\frac12) </m>. 
+              The region contains the entire area below the curve, lying above the horizontal line <m> y=0 </m>.
+              On the <m> x</m>-axis, the graph contains an arbitrarily chosen point <m> x </m>. 
+              The distance between the origin and the point <m> x </m> is given as the function <m> r(x) </m>, which will give us the radius of the cylindrical shell once the region is rotated about the <m> y</m>-axis.  
+              Additionally, coming up from the point <m> x </m> is a red vertical line, which ends after meeting the curve <m>y=1/(1+x^2)</m>. 
+              This vertical line is labelled <m>h(x)</m>, and will give the us the height of the cylindrical shell.
```
Comment (sean-fitzpatrick) [inline ptx/sec_shell_method.ptx]: For consistency with the rest of the book, we should use the American spelling 'labeled'.
(I don't like it, but it is what it is... Some day I'll make a Canadian edition.)

File: ptx/sec_shell_method.ptx [hunk 6]
```diff
@@ -1086,7 +1180,18 @@
...
-              <description></description>
+              <description>
+                <p>
+                  Graph of the region bounded by <m>y= \sin(x)</m> and the <m>x</m>-axis from <m>x=0</m> to <m>x=\pi</m>. 
+                  The leftmost point of the triangular region begins at the origin after which we see a singular sine wave ending at the point <m> (\pi,0) </m>. 
+                  The region contains the entire area to below the curve <m>y= \sin(x)</m>, lying above the <m>x</m>-axis.
+                  The graph contains an arbitrarily chosen value on the <m>x</m>-axis, labelled <m> x </m>, which is between <m> 0 </m> and <m> \pi</m>.
+                  The graph showcases the distance between the axis of rotation which is <m> x=0 </m> and the point labelled <m>x</m> as the function <m> r(x) </m>, which will be the radius of the cylindrical shells. 
+                  Additionally, coming up from the <m> x</m>-axis is a red vertical line, which meets the curve <m>y= \sin(x)</m> at the point <m>(x,\sin(x))</m>. 
+                  This vertical line is labelled <m>h(x)</m>, and will give the us the height of the cylindrical shells.
```
Comment (sean-fitzpatrick) [inline ptx/sec_shell_method.ptx]: labeled again

**Review by sean-fitzpatrick — CHANGES_REQUESTED** [review]
Great work! Minor changes only.

**Comment by supervladimir** [comment]
The errors should now be fixed.

**Review by sean-fitzpatrick — APPROVED** [review]

**Pull request merged**